content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+relatedImages:
+- image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  name: ""
+- image: docker.io/example/example-operator-bundle:0.2.0
+  name: ""
+- image: docker.io/example/example-operator:0.2.0
+  name: ""
+schema: olm.bundle
+
+```
+</details>
+
+## Semver Veneer
+
+Since a `veneer` is identified as an input schema which is processed to generate a valid FBC, we can define a `semver veneer` as a schema which uses channel conventions to facilitate the auto-generation of channels adherent to [Semantic Versioning](https://semver.org/) (semver) guidelines.  
+
+[**DISCLAIMER:** since version build metadata [MUST be ignored when determining version precedence](https://semver.org/#spec-item-10) when using semver, it cannot be used in any bundle included in the `semver veneer` and will result in a fatal error.]
+
+### Schema Goals
```

Comment:
What about we add all that is from each veener as a subsection inside instead of creating this other topic?
Example

## Veener A

### How does it work?
### When should I use it
### Schemas 


## Veener B

### How does it work?
### When should I use it
### Schemas

Reply:
I think there is sense in tweaking the information architecture. Personally, I find headers that are questions distracting in docs that aren't FAQs.

WDYT of something like:

- Basic veneer use-cases
- Basic veneer workflow
- Building a file-based catalog with the basic veneer (for the procedure)
- Example basic veneer
- Basic veneer reference (or schema)

Reply:
This approach to the headings would would also help with SEO